packages/debugger/style/sidebar.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .jp-DebuggerSidebar .jp-DebuggerSidebar-body {
-  height: 100%;
+  height: calc(100% - 24px);
```

Comment:
Could the 24px here and below be assigned to a variable to avoid the values diverging in the future (or someone being confused about the choice of 24px here specifically)?

Reply:
Fixed